frame/composable-traits/src/financial_nft.rs

```diff
@@ -30,6 +37,11 @@ pub trait FinancialNftProvider<AccountId>: Create<AccountId> + Mutate<AccountId>
 		version: &K,
 		nft_data: &V,
 	) -> Result<Self::InstanceId, DispatchError>;
+
+	fn split(
```

Comment:
Would be nice to have some docs at the trait level

Reply:
will be in NEXT pr for componding